app/resources/en/translation/addressLookup.json

```diff
@@ -6,11 +6,11 @@
         "message": "Please enter a postcode"
```

Comment:
This is not an optimisation and should be raised in a separate ticket

Reply:
Ok I'll revert this